setup.py

```diff
@@ -12,18 +12,21 @@
     python_requires=">=3.8",
     install_requires=[
         "python-dateutil",
-        "libcove2",
+        "libcove2>=0.2.1",
         "packaging",
         # Jsonschema 4.10 breaks the message
         #     'missingPersonType' is a dependency of 'missingPersonReason'
         # in tests/fixtures/0.1/badfile_all_validation_errors.json
-        "jsonschema<4.10",
+        "jsonschema==4.23.0",
```

Comment:
The above comment about 4.10 can come out now. 

Is it possible to lock to 4.23.x or >= 4.23? very specific locks in libraries can cause problems when someone tries to use them with other libs. (Same for next line)

Reply:
Changed to >= 4.23

Reply:
Can we sort jscc too?